docs/test_offline.md

```diff
@@ -1,7 +1,7 @@
 ## Test Offline Capabilities 
 
 - Create any mutation. (Procedure shown on previous page)
-- If your network is connected, then you will get a successful reponse from the server in ResponseCallback's `onSuccess()` method. You can perfrom UI Bindings here according to your needs.
-- If your network is not connected, you will get the mutation object in ResponseCallback's `onSchedule()` method. You can perform local UI Bindings here accordingly.
+- If your network is connected, then you will get a successful reponse from the server in ApolloCalback's `onResponse()` method. You can perfrom UI Bindings here according to your needs.
+- If your network is not connected, a toast will be shown regarding the same and your mutation would be successfully stored in an offline store.
```

Comment:
Does the library show this toast or is it the developer's job to do it?

Reply:
Oh yeah, developer's job. Removing it.